alerts/templates/email/task_fail.html

```diff
@@ -30,7 +30,7 @@
         <tr><td colspan="2" height="10"></td></tr>
         <tr>
           <td width="90%">
-            <a href="{{ .URL }}" style="font-family:Arial,sans-serif;font-weight:normal;font-size:13px;color:#006cbc" class="link">view logs</a>
+            <a href="{{$.Settings.Ui.Url}}/{{ .URL }}" style="font-family:Arial,sans-serif;font-weight:normal;font-size:13px;color:#006cbc" class="link">view logs</a>
```

Comment:
So you mean that it's currently the desired behavior that the URL in task failure emails not have a domain attached?